.husky/.huskyrc.json

```diff
@@ -0,0 +1,5 @@
+{
+ "hooks": {
+   "pre-commit": "npm run lint"
```

Comment:
It's the wrong config. Husky should run lint-staged and lint-staged will run the configured commands. You're running linter directly.

Reply:
Changed 'pre-commit' file and added info into 'package.json' file. But I'm still not sure that it was a great decision. Please, have a look at these files again!